Dockerfiles/manifests/cluster-agent/rbac-cluster-agent.yaml

```diff
@@ -0,0 +1,58 @@
+kind: ClusterRole
+metadata:
+  name: datadog-cluster-agent
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  - events
+  - endpoints
+  - pods
+  - nodes
+  - componentstatuses
+  verbs:
+  - get
+  - list
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  resourceNames:
+  - datadogtoken             # Kubernetes event collection state
+  - datadog-leader-election  # Leader election token
+  verbs:
+  - get
+  - update
+- apiGroups:  # To create the leader election token
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - create
```

Comment:
Why this `create` verb isn't within the block above ?
Are we creating other configmap than:
* datadogtoken
* datadog-leader-election

Reply:
This is not possible because resource names are not present in the URL for create, list, watch, and deletecollection API requests.